tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -9232,6 +9236,13 @@ node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1164,15):
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1169,24): error TS2304: Cannot find name 'KEY'.
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1169,30): error TS2304: Cannot find name 'VALUE'.
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1171,15): error TS2339: Property 'inverse' does not exist on type 'Map<any, any>'.
+node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1191,14): error TS2304: Cannot find name 'K'.
```

Comment:
These are new too.

Reply:
![image](https://user-images.githubusercontent.com/2932786/40207034-52a041b4-59e7-11e8-9178-abd8f24f82f4.png)
Also templates - this time templates for prototype methods not having the templates declared for the constructor (though they should).

Reply:
Yep, looks like the check I put in resolveEntityName needs to understand prototype assignments in addition to the prototype method assignments it already does.

Reply:
#24230